Gruntfile.js

```diff
@@ -79,7 +79,7 @@ module.exports = function( grunt ) {
 			},
 			test: {
 				src: [ "test/*.js", "test/functional/**/*.js", "test/unit/**/*.js",
-					"!test/config.js" ],
+					"test/compiler/**/*.js", "!test/config.js", "!test/compiler/_compiled/**" ],
```

Comment:
Will putting the generated files into `tmp/` affect the build/publish process? That's where grunt-contrib-uglify puts the minified files, but it appears they are not used after that.